Tasks/JavaToolInstallerV0/task.json

```diff
@@ -13,7 +13,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 193,
+        "Minor": 194,
```

Comment:
I am not sure if we should bump the task version in this PR since we are only adding new tests. Task itself doesn't have any changes.

Reply:
This branch based on https://github.com/microsoft/azure-pipelines-tasks/tree/users/v-aopareva/bug14214_fix_path_java_tool_installer - and this bump is on that PR